lib/ipmi_backend_utils.pm

```diff
@@ -262,6 +262,18 @@ sub adjust_for_ipmi_xen {
     }
 }
 
+# Set boot from pxe via ipmitool
+sub set_pxe_boot {
```

Comment:
This function has similarity with the following function "sub set_pxe_efiboot" . The latter seems to handle efiboot with efibootmgr and call handle multiple NIC pxe boot situation. I am wondering is it possible to merge/reuse that function? 
Besides, regarding set_pxe_boot function, can it survive on machines with two nic pxe boot items? And how about it on ipmi machines with different vendors, eg supermicro, dell, etc.

Reply:
set_pxe_efiboot can only be run on a machine installed OS, I guess. while set_pxe_boot control the BIOS directly. It can be run anytime, especially before boot up to install. They are used in different situations.

set_pxe_efiboot just configure BIOS via ipmitool. If the NIC boot order is unable to be configured by BIOS, the functioin can do nothing either, I think.

Reply:
I wonder whether the tests will work with replacing `boot_from_pxe` with `ipxe_install`. this block is exactly the same as https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/installation/ipxe_install.pm#L47. i am not sure what do you try to do then. If we treat the repo like this the maintenance and the squads collaboration becomes even harder. is it possible for a better approach?

Reply:
The test is supposed to work with both boot_from_pxe and ipxe_install with small adaptation

If ipxe_install would be used,  we need a http server which allows HTTP POST(like writer permission) similiar as:
`"IPXE_HTTPSERVER"="http://baremetal-support.qa.suse.de:8080"`

And config it in OSD/Beijing dhcp server the way like:
```
host unarmed-1  { hardware ethernet 18:C0:4D:86:23:A4; fixed-address 10.162.3.85; option host-name "unarmed-1"; filename "kernelqa/ipxe_aarch64.efi"; } 
```

And anything else?

@alice-suse, you may have some idea which one is more suitable to our virtualization tests.

Reply:
After discussion, we have a plan to move our test machines to use iPXE install, not limited to UEFI machines. We create a ticket to track this, poo#110133.

Hi @b10n1k, where is the http server or who owns it, "http://baremetal-support.qa.suse.de:8080"? Could our virtualization tests use it directly? our machines are in OSD and Beijing:
```
10.162.2.xx  //in OSD, including x86_64 & aarch
10.67.131.xx  //in Beijing VLAN
```

Reply:
Let me add some info here to avoid confusion. Yes, we have plan to move to ipxe install, but NOT NOW, because the beijing ipxe server is not ready for this yet. And julie has tasks blocked by missing this PR. So let's make the task continue with PR merged firstly, then we will switch to ipxe when all conditions are ready.

Reply:
@Julie-CAO  see https://confluence.suse.com/display/qasle/Baremetal+Support+Service it is server managed by kernel team